### ISSUE TITLE
Bioconda Package / Update README? 

### ISSUE THREAD

**Issue by apeltzer** [stschiff/sequenceTools#2]
Hi Stephan,

I just created a recipe for Bioconda for sequenceTools to make installation less work. Users can (once this is merged and build by the bioconda team) install your package this way:

`conda install sequencetools` (when they have an active bioconda channel)

You could also add a badge to your `README.md` to reflect this possibility - lots of people are using this nowadays and since you had already a new release out there, I thought this would be useful. 

e.g. having such a link here:

`[![Anaconda-Server Badge](https://anaconda.org/bioconda/sequencetools/badges/installer/conda.svg)](https://conda.anaconda.org/bioconda)`

(will be available when merged into `master` branch of bioconda). 

https://github.com/bioconda/bioconda-recipes/pull/9869

Cheers,
Alex

**Comment by stschiff** [comment]
Hi Alex,

great, thanks a lot! Please let me know once this is indeed merged, then I’ll add the badge!
Stephan

**Comment by apeltzer** [comment]
Its done: 

https://anaconda.org/bioconda/sequencetools/badges

You can also add a simple comment on your Readme on how to install with conda:

https://anaconda.org/bioconda/sequencetools

` conda install -c bioconda sequencetools `
should do the trick already ;-)

If you do a new release, you can either add it yourself or ask in the community (or me!) .

**Comment by stschiff** [comment]
Done. I've just added the badge. Thanks a lot!
